### ISSUE TITLE
Include Directive breaks due to filepath including the current file

### ISSUE THREAD

**Issue by taubenangriff** [xforce/anno1800-mod-loader#113]
https://github.com/xforce/anno1800-mod-loader/blob/8f5d693ab9c85e142f20acada6d12ab872d855e8/libs/xml-operations/src/xml_operations.cc#L430

At this point, mod_path seems to be the the path to the current file rather than the mod directory. 

I have an assets.xml that should reference assets_old.xml in the same directory. 

```XML
<ModOps>
    <Include File="data\config\export\main\asset\assets_old.xml" />
</ModOps>
```

which results in the following error: 

`[2021-11-07 16:03:21.105] [error] [[New Horizons] Main] Failed to parse E:\SteamLibrary\steamapps\common\Anno 1800\mods\[New Horizons] Main\data\config\export\main\asset\assets.xml\data\config\export\main\asset\assets_old.xml(1, 1): File was not found`

As you can see, it tries to look up the file starting at the directory of the current file the Include Directive was called from.

**Comment by jakobharder** [comment]
`xmltest` is also having inconsistent behavior. It uses the the working directory.

```text
test/
  assets-cheese.xml
  assets.xml
```

`<Include File="./assets-cheese.xml">` does work here, but only with `test` as the working directory - which is something I want to avoid.

**Comment by xforce** [comment]
Fixed in [2b4bac3](https://github.com/xforce/anno1800-mod-loader/commit/2b4bac3386db92155e78e312dae1d66dd0db16d1)
